### PR TITLE
HW-03 fix

### PR DESCRIPTION
**Pull request by Max-Pochepets**

### CODE REVIEW

**Review by KseniiaMakarova — APPROVED** [review]

File: src/main/java/mate/jdbc/dao/DriverDaoImpl.java [hunk 1]
```diff
@@ -17,9 +17,8 @@
 public class DriverDaoImpl implements DriverDao {
     @Override
     public Driver create(Driver driver) {
-        String query = "INSERT INTO drivers (driver_name, driver_license_number, "
-                    + "driver_login, driver_password) "
-                + "VALUES (?, ?, ?, ?)";
+        String query = "INSERT INTO drivers (name, number) "
```
Comment (boroda4436) [inline src/main/java/mate/jdbc/dao/DriverDaoImpl.java:20]: Can we inline these two lines?

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 2]
```diff
@@ -17,12 +17,14 @@
 public class ManufacturerDaoImpl implements ManufacturerDao {
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
-        String query = "INSERT INTO manufacturers (manufacturer_name,manufacturer_country) "
-                + "VALUES (?,?)";
+        String query = "INSERT INTO manufacturers (name, country) "
```
Comment (boroda4436) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:20]: same here

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 3]
```diff
@@ -57,11 +59,11 @@ public Manufacturer create(Manufacturer manufacturer) {
     public List<Manufacturer> getAll() {
         String query = "SELECT * FROM manufacturers WHERE deleted = FALSE";
         try (Connection connection = ConnectionUtil.getConnection();
-                Statement statement = connection.createStatement()) {
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
```
Comment (boroda4436) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:62]:
```suggestion
                PreparedStatement getAllManufacturersStatement = connection.prepareStatement(query)) {
```

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 4]
```diff
@@ -89,7 +92,7 @@ public Manufacturer update(Manufacturer manufacturer) {
 
     @Override
     public boolean delete(Long id) {
-        String query = "UPDATE manufacturers SET deleted = TRUE WHERE manufacturer_id = ?";
+        String query = "UPDATE manufacturers SET deleted = TRUE WHERE id = ?";
         try (Connection connection = ConnectionUtil.getConnection();
                 PreparedStatement preparedStatement = connection.prepareStatement(query)) {
```
Comment (boroda4436) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:97]:
```suggestion
                PreparedStatement deleteManufacturerStatement = connection.prepareStatement(query)) {
```

File: src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java [hunk 5]
```diff
@@ -36,15 +38,15 @@ public Manufacturer create(Manufacturer manufacturer) {
...
     public Optional<Manufacturer> get(Long id) {
-        String query = "SELECT manufacturer_id, manufacturer_name, "
-                + "manufacturer_country FROM manufacturers"
-                + " WHERE manufacturer_id = " + id + " AND deleted = FALSE";
+        String query = "SELECT * FROM manufacturers "
+                + "WHERE id = ? AND deleted = FALSE";
         try (Connection connection = ConnectionUtil.getConnection();
-                Statement statement = connection.createStatement()) {
-            ResultSet resultSet = statement.executeQuery(query);
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
```
Comment (boroda4436) [inline src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java:44]:
```suggestion
                PreparedStatement getManufacturerStatement = connection.prepareStatement(query)) {
```

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**